ige/CMakeLists.txt

```
@@ -122,12 +123,15 @@ set(sources ${sources}
     "src/plugin/input/Mouse.cpp"
     "src/plugin/InputPlugin.cpp"
     "src/plugin/InputPlugin.cpp"
+    "src/plugin/physics/bullet3/BulletGhostObject.cpp"
+    "src/plugin/physics/bullet3/BulletGhostObject.hpp"
     "src/plugin/physics/bullet3/BulletRigidBody.cpp"
     "src/plugin/physics/bullet3/BulletRigidBody.hpp"
     "src/plugin/physics/bullet3/BulletWorld.cpp"
     "src/plugin/physics/bullet3/BulletWorld.hpp"
     "src/plugin/physics/PhysicsWorld.cpp"
     "src/plugin/physics/RigidBody.cpp"
+    "src/plugin/physics/GhostObject.cpp"
     "src/plugin/PhysicsPlugin.cpp"
```

Keep the file list sorted alphabetically
```suggestion
    "src/plugin/InputPlugin.cpp"
    "src/plugin/physics/bullet3/BulletGhostObject.cpp"
    "src/plugin/physics/bullet3/BulletGhostObject.hpp"
    "src/plugin/physics/bullet3/BulletRigidBody.cpp"
    "src/plugin/physics/bullet3/BulletRigidBody.hpp"
    "src/plugin/physics/bullet3/BulletWorld.cpp"
    "src/plugin/physics/bullet3/BulletWorld.hpp"
    "src/plugin/physics/GhostObject.cpp"
    "src/plugin/physics/PhysicsWorld.cpp"
    "src/plugin/physics/RigidBody.cpp"
    "src/plugin/PhysicsPlugin.cpp"
```